libs/sdk-ui/src/base/localization/bundles/en-US.json

```diff
@@ -177,6 +177,21 @@
         "comment": "Means comparison of two values - for example '100 (this week) VERSUS 10 (last week)'.",
         "limit": 0
     },
+    "visualizations.headline.comparison.title.change": {
+        "value": "Change",
+        "comment": "The default label value of comparison when calculating as change.",
+        "limit": 0
+    },
+    "visualizations.headline.comparison.title.ratio": {
+        "value": "of",
+        "comment": "The default label value of comparison when calculating as ratio.",
```

Comment:
This requires bit more context for translators. Specify more what this "of" means please. You should be able to translate this to Vietnamese in the correct context without seeing the UI just with help of this text :)

Reply:
Updated!